syntax="proto3";

package demo4;

option cc_generic_services = true;

message HttpRequest { };
message HttpResponse { };

service HttpService {
    rpc Echo(HttpRequest) returns (HttpResponse);
};
